### DISCUSSION TITLE
Regarding Stripe [Saleor3.0]

### DISCUSSION THREAD

**Discussion by kdheeraz** [saleor/saleor#8060] [Q&A]
On Saleor 3.0 after enabling stripe payment gateway and providing a webhook, I am able to see the transaction being completed on Saleor and Stripe dashboard both.

But page I am getting after payment is 404 and there is no invoice_number as well.

With dummy gateway I am getting a Thank you page and Invoice pdf.

On Saleor Dashboard I can see the invoice but there is no invoice number.

please suggest if any issue in configuration or something else.

All three elements of Saleor are on 3.0 only.

**Comment by karolkielecki** [comment]
@kdheeraz Thanks for reporting. It seems there is something wrong on the storefront side. The dashboard creates the order, but the storefront can't handle it somehow. I've passed your case to the team. If we have time, we'll take a look at it. Which storefront version are you using?

**Reply by kdheeraz** [reply]
I am using 3.0 version of storefront (3.0.0-a.5).

Please have a look

Thanks Karol !!

**Reply by karolkielecki** [reply]
Could you please use the latest beta version https://github.com/mirumee/saleor-storefront/releases/tag/3.0.0-b.2?

**Reply by orzechdev** [reply]
Should be fixed in https://github.com/mirumee/saleor-storefront/pull/1098, @kdheeraz you may check out the latest master for now.
